lightdash/helm/lightdash/Chart.yaml

```diff
@@ -9,5 +9,5 @@ dependencies:
     version: 0.7.1
     repository: https://lightdash.github.io/helm-charts
   - name: postgres
-    version: 0.1.10
+    version: 0.1.17
```

Comment:
maybe need to update helm dependencies? (tgz and chart lock files)
also chart version